assets/js/app.js

```diff
@@ -2,7 +2,12 @@
  * Main entry file for app logic
  */
 
+// Config
+app.config = {

```

Comment:
Benaming van deze variabelen vind ik niet heel erg duidelijk. Wat configureert 'width_sm'? Misschien de `config` property hernoemen naar `breakpoints`?

Of hier geen statische properties van maken, maar een utility module die de breakpoints berekent adhv de base `em` font size van de browser.